app/src/main/java/com/example/myapplication/RecipeDetailActivity.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.myapplication
+
+import android.os.Bundle
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+
+class RecipeDetailActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_recipe_detail)
+
+        val recipeId = intent.getIntExtra("RECIPE_ID", -1)
```

Comment:
Nice usage of intent extras
Generally (not yet) you'd only pass the ID and allow that activity fetch that recipe for itself from database etc